src/Item.js

```diff
@@ -4,6 +4,33 @@ import React, { useEffect, useState } from 'react';
 import { Modal } from './components';
 import estimates from './lib/estimates';
 import { db } from './lib/firebase';
+import styled from 'styled-components';
+import { Checkbox } from '@material-ui/core';
+import { FaTrashAlt } from 'react-icons/fa';
+
+const ItemStyle = styled.ul`
+  text-align: left;
+  background-color: lightgray;
```

Comment:
I think we could increase the contrast by using the lighter shade of grey + shadow from Figma:
```
background: #F2F3F2;
box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.1), 0px 4px 8px rgba(0, 0, 0, 0.08), 0px 1px 12px rgba(0, 0, 0, 0.04);
```